src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.ts

```diff
@@ -176,24 +206,114 @@ closeDescription(): void {
         this.loggedIn = false;
     });
   }
-  
-  keyMaps(event: KeyboardEvent) {                
-    if (this.headerEditable || this.descEditable){           
-      this.save();
-      if (this.headerEditable) this.toggleHeader(); 
-      if (this.descEditable) this.toggleDescription();      
-    }
-  }
 
   preventDef(event: any) {
     event.preventDefault();
   }
 
-  resetEditables() {
+  activeSearchAssignee() {
```

Comment:
Do you think Assigning a user could be done any place else besides in the details view?